### PR TITLE
[Enhancement] [ALS-1792] Update creds and websocket implementation

### PR DESCRIPTION
**Pull request by wadhawh**
This PR has the following changes;

- Removes dependency of guest and auth creds from aws-amplify
- Removes dependency of websocket from aws-amplify
- Updates E2E workflow for simplicity
- Updates naming convention for services

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/core/AppWrapper/AppWrapper.tsx [hunk 1]
```diff
@@ -14,50 +12,14 @@ import "@demo/theme/appStyles.scss";
...
-	configureAmplify(amplifyConfig);
-
-	return (
-		<ThemeProvider theme={appTheme} nonce="dAnIsRazNonCe">
-			{children}
-		</ThemeProvider>
-	);
-};
+const AppWrapper: FC<AppWrapperProps> = ({ children }) => (
+	<ThemeProvider theme={appTheme} nonce="dAnIsRazNonCe">
```
Comment (mbalfour-amzn) [inline src/core/AppWrapper/AppWrapper.tsx:20]: Isn't the nonce supposed to be randomly generated on every request?

File: src/hooks/useCredsManager.ts [hunk 2]
```diff
@@ -3,150 +3,167 @@ import { useCallback, useEffect } from "react";
...
-					}, 10 * 60 * 1000);
-				} else {
-					/* If the credentials are not authenticated, set the refresh timeout */
-					timeout = setTimeout(() => {
-						clearCredsAndLocationClient();
-					}, differenceInMilliseconds(new Date(credentials.expiration || 0), new Date()));
-				}
+				timeout = setTimeout(() => {
+					refreshCredentials();
+				}, differenceInMilliseconds(new Date(credentials.expiration || 0), new Date()) - 300);
```
Comment (mbalfour-amzn) [inline src/hooks/useCredsManager.ts:80]: The previous logic didn't have the "- 300", why is that needed now?

Comment (its-aazizi) [reply src/hooks/useCredsManager.ts:80]: Credentials are refreshed 5 mins before expiration

File: src/core/AppWrapper/AppWrapper.tsx (hunk 1 above)
Comment (its-aazizi) [reply src/core/AppWrapper/AppWrapper.tsx:20]: It should be, I'll look into this. This was actually worked on by another developer at the start of this project. Even if we remove this, we will be good. I'll give that a test and let you know.

**Comment by its-aazizi** [comment]
@mbalfour-amzn raising another PR, since the PR I raised from [forked repo](https://github.com/aws-geospatial/amazon-location-features-demo-web/pull/243) shows conflicts

**Comment by its-aazizi** [comment]
@mbalfour-amzn this PR has been marked as stale and moved in draft. You can find the updated PR [here](https://github.com/aws-geospatial/amazon-location-features-demo-web/pull/245) that resolves the comments in this PR. This PR can be closed and branch deleted.
